src/background/service/ExtUtils.js

```diff
@@ -102,7 +102,7 @@ oasisExt.ext.ready({
                 const handled = oasis.signature.visitMessage(
                     {
                         withChainContext:
-                            ({
+                            /** @type {oasis.consensus.SignatureMessageHandlersWithChainContext} */ ({
```

Comment:
these create new type errors, since we're checking in strict mode. oasis-sdk itself doesn't use strict mode, so I hadn't noticed. I filed https://github.com/oasisprotocol/oasis-sdk/pull/790 over there